rubrik_cdm/cluster.py

```diff
@@ -606,3 +608,56 @@ def read_only_authorization(self, username, timeout=15):
 
         self.log("read_only_authorization: Granting read-only privilages to user '{}'.".format(username))
         return self.post("internal", "/authorization/role/read_only_admin", config, timeout)
+
+    def cluster_node_id(self,timeout=15):
+        """ Returns a list of node ids from all the nodes in the cluster.
+
+                Arguments:
+                    None
+
+                Keyword Arguments:
+                    None
+
+                Returns:
+                    dict -- The full API response from `POST /internal/node`.
+                """
+        self.log("cluster_node_id - Getting all the node ids from the from the cluster")
+        node_id = self.get('internal', '/node',timeout)
+        return node_id
+
+    def cluster_support_tunnel(self,enabled, timeout=15):
+        """ Function that can check the status, enable or disable the support tunnel.
+
+                        Arguments:
+                            enabled - with state of (True or False)
```

Comment:
We need to be more descriptive here. Something like "Flag to determine whether the support tunnel is enabled. Set to False to disable". Let's also make `True` the default value here. And since we have a default value that becomes a "Keyword Argument" so you can move it down there. Also, make sure to call out the fact that True is the default. Check some of the other docstrings for examples on what that looks like.